common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -209,6 +209,37 @@
                     success: function(response) {
                         var thread, discussionBreadcrumbsModel;
                         thread = new Thread(response.content);
+
+                        /* This is used as a signal to the Learning MFE to determine if the user
+                           should see the discussion milestone celebration.
+                           Note: at time of writing, this is only listened to for inline discussions
+                           inside of the Learning MFE. Discussion Tab posts will happen once the
+                           Discussion MFE exists */
```

Comment:
This was a little confusing to me. Does my interpretation reflect the current situation?

"At time of writing, this is only listening for discussions that are posted within the courseware section of the Learning MFE, and NOT for discussions that are posted within the Discussions tab. There are plans to replace the Discussions tab with an MFE. Once the Discussions MFE is built, we will have to update this code to account for both posts within the courseware and within the Discussions tab."

Reply:
I feel like this comment conflicts with what you say in the comment below..

First you say:
> Discussion Tab posts will happen once the Discussion MFE exists

(What's meant by "will happen"?)

And then below you say:
>We do this here rather than in the Learning MFE (where the celebration
                           actually happens right now) to catch both inline discussions and posts on
                           the discussion tab.

I interpreted this as "We are not accounting for the discussion tab... but we are doing this here to account for the discussion tab"